applications/tari_dan_wallet_daemon/src/jrpc_server.rs

```diff
@@ -46,6 +46,17 @@ pub async fn listen(
 
 async fn handler(Extension(context): Extension<Arc<HandlerContext>>, value: JsonRpcExtractor) -> JrpcResult {
     info!(target: LOG_TARGET, "🌐 JSON-RPC request: {}", value.method);
+    if value.method == "rpc.discover" {
+        return Ok(JsonRpcResponse::success(
+            value.id,
+            serde_json::from_str::<HashMap<String, Value>>(include_str!("../openrpc.json")).map_err(|e| {
```

Comment:
Think this can just be a Value
```suggestion
            serde_json::from_str::<Value>(include_str!("../openrpc.json")).map_err(|e| {
```

Reply:
Nah, I tried that, but it automatically chooses an int or something and fails on deserialize

Reply:
Basically got this error https://stackoverflow.com/questions/48595735/invalid-type-map-expected-a-sequence-when-deserializing-a-nested-json-struct

Reply:
Very strange